llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgTransform/TransformInterpreterPassBase.h

```diff
@@ -111,21 +148,43 @@ class TransformInterpreterPassBase : public GeneratedBase<Concrete> {
   /// fails.
   LogicalResult runAfterInterpreter(Operation *) { return success(); }
 
+  /// Hook for passes to indicate payload roots for successive invocations of
+  /// the transform dialect interpreter.
+  void getPayloadRoots(SmallVectorImpl<Operation *> &targets) {
```

Comment:
Why not just `return SmallVector<Operation*>` ?

Reply:
This is the preferred way in LLVM/MLIR and the code here is intended to go there.